nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -179,13 +179,40 @@ def _get_stat_collector(self, quantizer_config: QuantizerConfig) -> TensorStatis
                                                                         num_samples=self._parameters.number_samples)
         raise RuntimeError('This range type is not supported!')
 
+    def _filter_nncf_graph(self, nncf_graph: NNCFGraph) -> NNCFGraph:
+        """
+        Removes all Constant nodes from NNCFGraph, making it inference graph.
```

Comment:
What is the problem with building the inference graph in `NNCFGraphFactory.create(model)`? It looks like method removes Constant nodes, but not Constant subgraphs.

Reply:
`MinMaxQuantization` should use filtered `NNCFGraph` without Constant subgraphs to avoid incorrect q/dq placements. At the same time "full" `NNCFGraph` could be useful for other algorithms. Also, it has more nice visualization with Constant subgraphs, e.g. weight quantizers.

Reply:
@kshpv Could you please look at my comment above?